.vscode/settings.json

```diff
@@ -18,4 +18,7 @@
       "editor.defaultFormatter": "JuanBlanco.solidity"
   },
   "solidity.compileUsingRemoteVersion": "v0.8.19+commit.7dd6d404"
+  "[markdown]": {
```

Comment:
Sorry about this - should have gotten the exclusions into the original formatting PR